tasks/eth/base-001-MCP-L1/README.md

```diff
@@ -0,0 +1,40 @@
+# Base Mainnet MCP L1 Upgrade
+
+Status: [READY TO SIGN]()
+
+## Objective
+
+This is the playbook for executing Upgrade #6, Base Mainnet Multi-Chain Prep (MCP) L1.
+
+The proposal was:
+
+- [X] Posted on the governance forum here: https://gov.optimism.io/t/upgrade-proposal-6-multi-chain-prep-mcp-l1
+- [X] Approved by Token House voting here: https://vote.optimism.io/proposals/47253113366919812831791422571513347073374828501432502648295761953879525315523
+- [X] Not vetoed by the Citizens' house here: https://snapshot.org/#/citizenshouse.eth/proposal/0x2bc6565053b73813c6b0a001c8c07eb5656234b4d8bae12ba6541250993d1d25
+- [X] Executed on OP Sepolia: https://github.com/ethereum-optimism/superchain-ops/tree/6ea64b21416917e3dfcb867fd0c76db90fb46277/tasks/sep/003-MCP-L1
+- [X] Executed on OP Mainnet: https://github.com/ethereum-optimism/superchain-ops/tree/11bd3a24b3c18ccf637e32182045a377a89ec22f/tasks/eth/006-MCP-L1
+- [X] Executed on Base Sepolia: https://sepolia.etherscan.io/tx/0xdd0716a31a2b71f0f4162f474bbfb3873eb570e0395d42f3a945b65e35562dbf
+
+The governance proposal should be treated as the source of truth and used to verify the correctness
+of the onchain operations.
+
+## Simulation
+
+Please see the "Simulating and Verifying the Transaction" instructions in [SINGLE.md](../../../SINGLE.md).
```

Comment:
This is a nested safe, since the owner safe of `0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c` is a 2/2 where each owner is a safe. We'll need to change all references of Single to Nested. Non-comprehensive list:
- In this file
- The CI file should invoke `nested.just` once with `just {snip} simulate foundation` and again with `just {snip} simulate council` 
- Renaming the `SignFomJson.s.sol` to `NestedSignFromJson.s.sol`
- Changing `import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";` to `import {NestedSignFromJson as OriginalSignFromJson} from "script/NestedSignFromJson.s.sol";`